backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/ad/persistence/modelgen/ModelClassGenerator.java

```diff
@@ -532,4 +532,10 @@ private void createImports(final StringBuilder sb)
 		}
 		sb.append(NL);
 	}
+
+	private static boolean skipColumn(@NonNull final ColumnInfo columnInfo)
```

Comment:
pls rename it to `isSkipColumn` to make it obvious for developer.

about not generating RestAPI Custom Columns, wondering if there is a reason behind?
I mean, i am not totally left or right but i have the feeling that it shall not matter.
We have other ways to control if a column is generated, i.e.
* entity type is matching
* AD_Column.IsForceIncludeInGeneratedModel

wdyt?

cc @metas-ts

Reply:
well, I thought those custom columns should not be part of the model primarily bc if they would be part of the model then they are not really custom anymore, but yeah if you think it doesn't make sense we can let them be :-?

Reply:
just loud thinking... I am still not convinced, not my arguments nor yours.
so leave it as you did it.